plugins/woocommerce/package.json

```diff
@@ -703,7 +715,6 @@
 		"@woocommerce/api": "workspace:*",
 		"@woocommerce/e2e-utils-playwright": "workspace:*",
 		"@woocommerce/eslint-plugin": "workspace:*",
-		"@woocommerce/woocommerce-rest-api": "^1.0.1",
```

Comment:
Is it feasible to drop it from blocks package dependencies as well?

Reply:
Not sure, I only worked on removing it from core. 
Doing a quick search it looks like it's not used in blocks either. 
I'll try a remove it and see if breaks anything.

Reply:
Thanks for trying!

Reply:
Opened a new PR, I messed something up in this one: https://github.com/woocommerce/woocommerce/pull/56203
CI passed, I think we're good.